pkgs/development/python-modules/flask-simpleldap/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, flask
+, python-ldap
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "flask-simpleldap";
+  version = "2.0.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "alexferl";
+    repo = pname;
```

```suggestion
    repo = "flask-simpleldap";
```

We consider this an anti-pattern, since pname is overridable

---

Checking for outdated packages for the 24.11 release cycle I noticed I never changed this, sorry! It's fixed now.